src/sources/dune.py

```diff
@@ -1,5 +1,7 @@
 """Source logic for Dune Analytics."""
 
+from __future__ import annotations
```

Comment:
What was this for?

Reply:
Honestly, no idea. It was already there, or potentially added by one of our very clever linters/fixers.